Assets/Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -154,10 +154,7 @@ protected void SendTargetRPCInternal(NetworkConnection conn, Type invokeClass, s
                 return;
             }
             // connection parameter is optional. assign if null.
-            if (conn == null)
-            {
-                conn = connectionToClient;
-            }
+            conn = conn ?? connectionToClient;
```

Comment:
not sure about that one. do you really think this is more simple?
I generally like shorter code, but for the old one it's a straight forward thought of 'if it's null then assign'. with your change, it's 'what is that ?? thing again? oh that's right, assign to self but if self is null then assign to the thing on the right'.